### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_QueueTable_ScreenReader: NVDA is announcing whole table data in one go after navigating to each cell of the table.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3479]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.3.1;#Win32;#A11ySev2;#Benchmark;#NVDA;#JAWS;#FTP;#STP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen Reader : JAWS(2020.2006.12ILM),NVDA(2020.2)

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Navigate to queue table cell. 
7. Check if NVDA is announcing whole table data in one go after navigating to each cell of the table after navigating using arrow key.

### **Actual:**
NVDA is announcing whole table data in one go after navigating to each cell of the table.

This issue is also observed with **JAWS**

Similar issue also observed for following scenarios:
1. Disk-->DiskInfoTable

### **Expected:**
NVDA should announce only the cell value of the table along with associated row and column after navigating to each cell of the table.

### **User Impact:**
Screen reader users will not get the correct value of the cell and get confused.

### **Attachment:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-prraha_microsoft_com/EQ5hD11vS_9OgzqfM_KiQ7gBi8RDAhrOqF9iJSM0StmCPA?e=Hj6GGW

**Comment by craxal** [comment]
@pranshu512 I am unable to reproduce the issue with NVDA or VoiceOver. The screen reader does not announce the entire contents of the table, just the contents of the currently selected cell as well as the number of items displayed. Please verify again. If the bug is still reproducing, please update the bug with more details.

**Comment by pranshu512** [comment]
I verified this issue on version 1.16.0-dev and it is not repro. Hence, Closing this bug

[3479_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/5407626/3479_Closed.zip)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
